webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvValidationResult.java

```diff
@@ -1,13 +1,14 @@
 package org.opentestsystem.rdw.admin.model;
 
 /**
- * This model represents a CSV Validation failure containing the failing row
- * and failure message.
+ * This model represents a CSV Validation result containing the ending row
+ * and success or failure message
  */
-public class CsvValidationFailure {
+public class CsvValidationResult {
 
     private long row;
     private String message;
+    private boolean isOk;
```

Comment:
Nit: Generally in java beans the property is named "ok" and the accessor is named isOk()